Tests/DatadogTests/Datadog/Mocks/RUMDataModelMocks.swift

```diff
@@ -229,6 +229,7 @@ extension RUMErrorEvent: RandomMockable {
                     url: .mockRandom()
                 ),
                 source: [.source, .network, .custom].randomElement()!,
+                sourceType: .ios,
```

Comment:
This is randomized mock (`mockRandom()`) for fuzzy testing, so it should use random value (like `source:`).